pom.xml

```diff
@@ -90,6 +91,21 @@
 
     <dependencies>
         <!-- Main dependencies that use a BOM so no explicit versions needed -->
+        <dependency>
```

Comment:
given the above comment, these new deps should be moved down to where deps are declared with specific versions.

Reply:
Fair point - fixed.